plugins/by-name/chadtree/default.nix

```diff
@@ -502,11 +512,21 @@ in
       };
     in
     mkIf cfg.enable {
-      extraPlugins =
-        [ cfg.package ]
-        ++ (optional (
-          cfg.iconsPackage != null && (cfg.theme == null || cfg.theme.iconGlyphSet == "devicons")
-        ) cfg.iconsPackage);
+      # TODO: added 2024-09-20 remove after 24.11
+      plugins.web-devicons.enable = mkOverride 1490 true;
+      warnings =
+        optional
+          (
+            (cfg.theme == null || cfg.theme.iconGlyphSet == "devicons")
+
+            && options.plugins.web-devicons.enable.highestPrio == 1490
```

Comment:
Again, this warning has an `&&` check, but we define the enable option unconditionally above. Am I missing something?